app/views/start-application.njk

```diff
@@ -44,4 +45,62 @@
         </form>
       </div>
     </div>
+
+    {% if agreements.length > 0 %}
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full">
+        <hr class="govuk-section-break govuk-section-break--xl govuk-section-break--visible">
+        <table class="govuk-table">
+          <caption class="govuk-table__caption govuk-table__caption--m">Applications</caption>
+          <thead class="govuk-table__head">
+            <tr class="govuk-table__row">
+              <th scope="col" class="govuk-table__header">Application</th>
+              <th scope="col" class="govuk-table__header">Last updated</th>
+              <th scope="col" class="govuk-table__header">Status</th>
+              <th scope="col" class="govuk-table__header"></th>
+            </tr>
+          </thead>
+          <tbody class="govuk-table__body">
+            {% for agreement in agreements|sort(true, false, 'updatedAtFormatted') %}
+            <tr class="govuk-table__row">
+              <td class="govuk-table__cell">{{ agreement.agreementNumber }}</td>
+              <td class="govuk-table__cell">{{ agreement.updatedAtFormatted }}</td>
+
+              {% if agreement.status.status === 'In Progress' %}
+                <td class="govuk-table__cell">
+                  <strong class="govuk-tag govuk-tag--yellow"> {{ agreement.status.status }} </strong>
+                </td>
+              {% elif agreement.status.status === 'Submitted' %}
+                <td class="govuk-table__cell">
+                  <strong class="govuk-tag govuk-tag--blue"> {{ agreement.status.status }} </strong>
+                </td>
+              {% else %}
+                 <td class="govuk-table__cell">
+                  <strong class="govuk-tag govuk-tag--grey"> {{ agreement.status.status }} </strong>
+                </td>
+              {% endif %}
+
+              <td class="govuk-table__cell">
+                {% if agreement.status.status === 'In Progress' %}
+                  <form method="POST" action="/start-application" novalidate>
+                    <input type="hidden" name="crumb" value="{{crumb}}"/>
```

Comment:
consider `value="{{ crumb }}"` for consistency

Reply:
changed